core/src/main/kotlin/com/alecstrong/sql/psi/core/mysql/MySql.bnf

```diff
@@ -216,4 +220,17 @@ drop_table_stmt ::= DROP TABLE [ IF EXISTS ] [ {database_name} '.' ] {table_name
   implements = "com.alecstrong.sql.psi.core.psi.SqlDropTableStmt"
   override = true
   pin = 2
-}
\ No newline at end of file
+}
+
+insert_stmt ::= [ {with_clause} ] INSERT [ 'LOW_PRIORITY' | 'DELAYED' | 'HIGH_PRIORITY' ] [ 'IGNORE' ] [ INTO ] [ {database_name} '.' ] {table_name} [ AS {table_alias} ] [ '(' {column_name} ( ',' {column_name} ) * ')' ] {insert_stmt_values} [ ON 'DUPLICATE' KEY UPDATE assignment_list ] {
```

Comment:
is it supposed to be `'OR' 'IGNORE'` here?

Reply:
Nope, for MySql it's legit `INSERT IGNORE INTO table`.

```
INSERT [LOW_PRIORITY | DELAYED | HIGH_PRIORITY] [IGNORE]
    [INTO] tbl_name
    ...
```

Taken  from https://dev.mysql.com/doc/refman/8.0/en/insert.html